### DISCUSSION TITLE
[@types/react-dual-listbox] Updating [Addition] property: Required

### DISCUSSION THREAD

**Discussion by TeoD3v** [DefinitelyTyped/DefinitelyTyped#63697] [Issues with a @types package] [answered]
Please add this one.

    /**
     * 	If true, this component will require selected to be non-empty to pass a form validation
     *
     * @default false
     */
    required?: boolean;
    /**

[a comment by an automated account is not shown]

**Comment by wKovacs64** [comment, marked as answer]
I've removed myself as maintainer as I no longer use this library due to lack of maintenance (not to mention I didn't write the original types). Ironically, the library has recently received a slew of updates and is now several major versions ahead of where it was when the current types were published. Barring the library including its own types, someone will need to take up the mantle and update DT accordingly.
